tests/test_adaptor_increase_maxorder.cc

```diff
@@ -117,21 +117,21 @@ namespace rascal {
         std::cout << "position: " << atom.get_position() << std::endl;
       }
 
-      for (auto pair : atom) {
+      for (auto pair : atom.pairs()) {
         npairs++;
         if (verbose) {
           std::cout << "   complete pair " << atom.back() << " " << pair.back()
                     << " glob " << pair.get_global_index() << std::endl;
         }
-        for (auto triplet : pair) {
-          n_triplets++;
-          if (verbose) {
-            std::cout << "             triplet " << triplet.back() << " global "
-                      << triplet.get_global_index() << std::endl;
-            std::cout << "                         complete " << atom.back()
-                      << " " << pair.back() << " " << triplet.back()
-                      << std::endl;
-          }
+      }
+      for (auto triplet : atom.triplets()) {
```

Comment:
Hehe, almost got the indentation wrong, but of course, we can now directly iterate over triplets.